assets/js/modules/ads/components/common/WooCommerceRedirectModal.js

```diff
@@ -114,11 +128,19 @@ export default function WooCommerceRedirectModal( {
 	const { navigateTo } = useDispatch( CORE_LOCATION );
 	const { dismissNotification } = useDispatch( CORE_NOTIFICATIONS );
 
-	const handleGoogleForWooCommerceRedirect = useCallback( () => {
+	const handleGoogleForWooCommerceRedirect = useCallback( async () => {
 		if ( ! isAccountLinkedViaGoogleForWoocommerceNoticeDismissed ) {
 			dismissNotification( 'account-linked-via-google-for-woocommerce' );
 		}
 
+		if ( ! isGoogleForWooCommerceAdsConnected ) {
+			await trackEvent(
+				`${ viewContext }_pax_wc-redirect`,
+				'choose_gfw',
+				trackEventLabel
+			);
+		}
```

Comment:
`choose_gfw` should be tracked in both cases where the user chooses not to use SK but it seems to only be tracked in the once case where GFW is not already connected

Reply:
Fixed in my latest iteration, thanks @aaemnnosttv